minesweeper.py

```diff
@@ -131,6 +131,7 @@ def generate_random_mine_locations(self, where_user_clicked):
                 self.mine_locations.append((x, y))
 
     def update_statusbar_mines_left(self):
+        """ Prints out how many mines are left when flagging squares """
```

Comment:
This method isn't only for flagging squares:

```suggestion
        """ Prints out how many mines are left """
```